javascript/swiggy.js

```diff
@@ -0,0 +1,158 @@
+const pdpModule = (function() {
+    function fetchCategories() {
```

Comment:
`fetchCategories`, `fetchMenuItems`, `fetchCartItems` - these should be outside the module. These are simple functions which return an output and don't expose any core data points globally. Also, keeping them out will allow you to reuse them in other modules later, if required